src/mocks.js

```diff
@@ -175,6 +175,61 @@ export const routes = {
   ],
 };
 
+export const patterns = {
+  setUpSuccessfulMock: (client) => {
+    const listResponse = () => new Response(
+    Client.toBlob(patterns.list),
+      {
+        headers: {
+          Link: '</1/SYNC/patterns?page=1&perPage=10&q=blue&sort=>; rel="next", </1/SYNC/patterns?page=1&perPage=10&q=blue&sort=>; rel="last"',
+        },
+      });
+    const singleResponse = () => new Response(Client.toBlob(patterns.getById(1)));
+
+    fetchMock
+      .get(client.resolve('/1/SYNC/patterns?[age=1&perPage=10&q=blue&sort='), listResponse)
+      .get(client.resolve('/1/SYNC/patterns/1'), singleResponse);
+  },
+  getById: id => patterns.list.find(p => p.id === id),
+  list: [
+    {
+      id: 1,
+      route: {
+        href: '/1/SYNC/routes/140',
```

Comment:
I think this should be `/1/SYNC/routes/1` to stay consistent with the rest of the mocks